lib/task.js

```diff
@@ -283,6 +286,8 @@ export class Task {
     this.taskState = 'pending';
     this.options = options;
 
+    this.queue = this.createQueue(this.claim.credentials, runtime);
```

Comment:
Ignore this, because we have a Reclaimer object per task to claim, the queue instance we use should be for that specific claim, so not on task.queue itself.